frontend/src/ui/components/Company/Assessments/CreateAssessment/SelectType.jsx

```diff
@@ -78,6 +78,8 @@ const SelectType = () => {
   }, [formData, touched]);
 
```

Comment:
this is not forme to merge tho.... pls remove... this belongs to FE

Reply:
@dasaint001 Invite the FE lead to look at the aspect he is to merge.

Reply:
it's someone from frontend that added it, if I remove it it'll cause conflict, it was added earlier when i was resolving conflict, part of the incoming change

Reply:
and merging this can also cause conflicts in the FE folder...

Reply:
Please if you have anything to do in the frontend folder, create a separate PR for that...